test/ng/compileSpec.js

```diff
@@ -5949,6 +5950,37 @@ describe('$compile', function() {
       });
 
 
+      it('should not merge text elements from transcluded content', function() {
+        module(function() {
+          directive('foo', valueFn({
+            transclude: 'content',
```

Comment:
in the old days, `transclude: 'content'` is what now is called `transclude: true`. Many tests still keep this old nomenclature